ext/webextension/src/browser_action/main_popup.js

```diff
@@ -509,4 +509,40 @@ document.querySelector('#siteconfig_show').addEventListener('click', function()
     ui.hide('#siteconfig_show');
 });
 
+function indetifierUpdate()
+{
+    var text = document.querySelector("#username").value;
+    text += document.querySelector('#masterkey').value;
+    var identifier = document.querySelector('#identifier');
+    if (text == '' || text == null || text.length < 6)
```

Comment:
I think (hope :))  all the rest of the code use K&r style braces (ie not on the next line). Would be nice to keep it consistent

Reply:
I have updated the code as requested.